selfdrive/ui/qt/offroad/networking.cc

```diff
@@ -44,9 +45,19 @@ std::string exec(const char* cmd) {
 // Networking functions
 
 Networking::Networking(QWidget* parent, bool show_advanced) : QWidget(parent){
-  try {
-    wifi = new WifiManager(this);
-  } catch (std::exception &e) {
+  bool connected = false;
+  for(int i = 0 ; i < 20 ; i++) {
```

Comment:
doesn't this delay startup? I'd just move the wifi init to another function and try to init on the refresh callback if we haven't yet